docs/user-guide/builtins.md

```diff
@@ -571,33 +571,54 @@ from pyscript import sync
 sync.hello("PyScript")
 ```
 
-### `pyscript.py_modules`
+### `pyscript.py_import`
 
 !!! warning
 
     **This is an experimental feature.**
 
     Feedback and bug reports are welcome!
 
-If you have a lot of packages referenced in your configuration, startup
+If you have a lot of Python packages referenced in your configuration, startup
 performance may be degraded as these are downloaded.
 
-If a package is only needed under certain circumstances, we provide an
+If a Python package is only needed under certain circumstances, we provide an
 asynchronous way to import packages that were not originally referenced in your
 configuration.
 
-```html title="a pyscript.py_modules example"
+```html title="A pyscript.py_import example."
 <script type="py" async>
-from pyscript import py_modules
+from pyscript import py_import
 
-matplotlib, regex, = await py_modules("matplotlib", "regex")
+matplotlib, regex, = await py_import("matplotlib", "regex")
 
 print(matplotlib, regex)
 </script>
 ```
 
-The `py_modules` call returns an asynchronous tuple containing the modules
-referenced as string arguments.
+The `py_import` call returns an asynchronous tuple containing the Python
+modules provided by the packages referenced as string arguments.
+
+### `pyscript.js_import`
+
+If you have a lot of JavaScript modules referenced in your configuration,
+startup performance may be degraded as these are downloaded.
+
+If a JavaScript module is only needed under certain circumstances, we provide
+an asynchronous way to import packages that were not originally referenced in
+your configuration.
+
+```html title="A pyscript.js_import example."
+<script type="py" async>
+from pyscript import js_import, window
+
+escaper, = await js_import("https://esm.run/html-escaper")
+
+window.console.log(escaper)
+```
+
+The `js_import` call returns an asynchronous tuple containing the JavaScript
```

Comment:
array ... to be precise, or a JS Array proxy ... although tuple works for Python devs :-)